src/api/cvc4cpp.h

```diff
@@ -2254,6 +2256,129 @@ struct CVC4_EXPORT RoundingModeHashFunction
   inline size_t operator()(const RoundingMode& rm) const;
 };
 
+/* -------------------------------------------------------------------------- */
+/* Statistics                                                                 */
+/* -------------------------------------------------------------------------- */
+
+/**
+ * Represents a snapshot of a single statistic value.
+ * A value can be of type int64_t, double, std::string or a histogram
+ * (`std::map<std::string, uint64_t>`).
+ * The value type can be queried (using `isInt`, `isString`, etc.) and
+ * the stored value can be accessed (using `getInt`, `getString`, etc.).
+ */
+class Stat
```

Comment:
Should this be exported via `CVC4_EXPORT`?

Reply:
Yes, I guess `Stat`, `Statistics` and the corresponding `operator<<` should all be `CVC4_EXPORT`.